pkg/drivers/kvm/domain.go

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
@dlorenc @aaron-prindle Where should the in-tree drivers live?  I just used the location that @dlorenc had in his hyperkit PR.  We have the none driver in `pkg/minikube/machine/drivers/none`